extensions/2.0/Khronos/KHR_spz_compression/README.md

```diff
@@ -0,0 +1,208 @@
+
+| Splat Data | glTF Attribute | Accessor Type | Component Type | Required |
+| --- | --- | --- | --- | --- |
+| Position | POSITION | VEC3 | float | yes |
+| Color (Spherical Harmonic 0 (Diffuse)) | COLOR_0 | VEC3 or VEC4 | unsigned byte or float normalized | yes |
+| Alpha | _OPACITY | SCALAR | float or unsigned byte | no |
+| Rotation | _ROTATION | VEC4 | float | yes |
+| Scale | _SCALE | VEC3 | float | yes |
+| Spherical Harmonics | COLOR_1 - COLOR_15 | VEC4 | float | no |
+
+If `_OPACITY` is omitted, see [Implementation](#implementation) below.
+
+#### SPZ header properties embedded in the compressed data stream
+
+The following properties are within the SPZ stream header. They are exposed here to be informative and facilitate decoder setup prior to any decompression occurring.
+
+##### numPoints
+
+The number of Gaussian splats in the compressed data.
+
+##### fractionalBits
+
+The number of fraction bits to use for `POSITION` quantization.
+
+##### shDegree
+
+The degree of spherical harmonics contained in the compressed data. This can range from 0 to 3.
+
+##### flags
+
+A set of bit flags reserved for future use. Currently it is only used for an anti-aliasing flag.
+
+### Accessors
+
+```json
+  "accessors": [{
+      "componentType": 5126,
+      "count": 590392,
+      "type": "VEC3"
+    }, {
+      "componentType": 5126,
+      "count": 590392,
+      "type": "SCALAR"
+    }, {
+      "componentType": 5121,
+      "count": 590392,
+      "type": "VEC3"
+    }, {
+      "componentType": 5126,
+      "count": 590392,
+      "type": "VEC3"
+    }, {
+      "componentType": 5126,
+      "count": 590392,
+      "type": "VEC4"
+    }],
+```
+
+At minimum accessors will be defined for `POSITION`, `COLOR_0`, `_ROTATION`, and `_SCALE`. Each must have `componentType`, `count`, and `type` defined.
+
+Accessor `type` is defined for the resulting type after decompression and dequantization has occurred.
+
+The accessor `count` must match `numPoints` in `KHR_spz_compression`.
+
+## Conformance
+
+The recommended process for handling SPZ compression is as follows:
+
+- If the loader does not support `KHR_spz_compression`, it must fail.
+- If the load does support `KHR_spz_compression` then:
+  - The loader must process `KHR_spz_compression` data first. The loader must get the data from `KHR_spz_compression`'s `bufferView` property.
+  - The loader then must process `attributes` of the `primitive`. When processing the loader must ignore any `bufferView` and `byteOffset` in the `accessor` and instead use values derived from the decompressed data streams. This data can be used to populate the `accessors` or render directly.
+  - Any attributes not listed in `KHR_spz_compression`'s `attributes` but are listed in the `primitive` must then be processed by the loader.
+
+## Implementation

```

Comment:
We should include all of the relevant stuff from our implementation section of the last proposal, including how to sort the splats and rasterize them.